### ISSUE TITLE
Product Filters Redesign > Price Filter: Update block styles settings

### ISSUE THREAD

**Issue by thealexandrelara** [woothemes/woocommerce#51821]
# Description

The Active block informs shoppers which product filters are active and lets them easily clear them. As part of the work to redesign the Product Filters blocks, we need to update the design of the Active filters block to the new design.

# Dependencies

> [!IMPORTANT]
> This issue is blocked by #51622 

# Acceptance Criteria

## Scenario 1: Merchants can edit the Typography, Dimensions, and Border settings from the Inspector Controls
_**Given**_ a merchant adds the Product Filter block to a page/template
_**When**_ they select the Price Filter block
_**Then**_  the Typography, Dimensions, and Border settings are shown within the Inspector Controls [Image Reference 1]
_**and**_  they can change the typography, dimensions, and border settings
_**and**_  the changes are reflected on the canvas

| Reference: Typography, Dimensions, and Border settings in the Inspector Controls |
|--------|
| ![CleanShot 2024-10-01 at 17 19 05@2x](https://github.com/user-attachments/assets/7d6dceea-f8e6-4f3a-b81f-3d23a262b5e9) | 

---

## Scenario 2: Shoppers can see customized Typography, Dimensions, and Border on the block
_**Given**_ a merchant adds the Product Filter block to a page/template
_**and**_  they edit the typography, dimensions and border settings
_**When**_ a shopper visits a page containing the customized Price Filter
_**Then**_  the customizations made by the merchants is correctly displayed.

---

**Comment by dinhtungdu** [comment]
Note that slider handle color settings should belong to the Price Slider block, not the Price filter block. For filter blocks, we only provide basic color settings like color and background. Settings that are related to a specific display style like slider here belong to the Price Slider block.

**Comment by thealexandrelara** [comment]
> Note that slider handle color settings should belong to the Price Slider block, not the Price filter block. For filter blocks, we only provide basic color settings like color and background. Settings that are related to a specific display style like slider here belong to the Price Slider block.

Exactly. That's why the acceptance criteria only consider the "Typography, Dimensions, and Border settings." The screenshot is from the original Figma design, where the Color settings were expected, but they are not an expectation for this issue.

**Comment by retrofox** [comment]
![Image](https://github.com/user-attachments/assets/535904ab-73df-43b6-92ee-41359aec2933)
Should we add an option to set that color?

**Comment by dinhtungdu** [comment]
> Should we add an option to set that color?

That's a design question so I'm tagging @j111q. Jill, can you guide us here? Note that we have settings for the handle, handle border, and slider.

**Comment by thealexandrelara** [comment]
I have another question regarding the permitted customizations. I noticed that when the mouse hovers over the thumb, or the thumb is selected, its color changes to the default theme color. Should we allow users to customize the hover color?

https://github.com/user-attachments/assets/17c62155-043f-4bef-a58f-a634b3f173a9

**Comment by dinhtungdu** [comment]
> Should we add an option to set that color?

@retrofox what do you think about reusing the Slider color with opacity for that color.

**Comment by retrofox** [comment]
Do you mean hardcoding the style, applying simple opacity, let's say 0.7?

**Comment by retrofox** [comment]
If so, it seems like a good, quick solution. We can iterate later. Adding a new pair of slider and border options just for the hover status seems like too much.

**Comment by dinhtungdu** [comment]
> Do you mean hardcoding the style, applying simple opacity, let's say 0.7?

@retrofox No hardcoding. I mean using the Slider color setting. The green color in your example in https://github.com/woocommerce/woocommerce/issues/51821#issuecomment-2451635209.

**Comment by retrofox** [comment]
> > Do you mean hardcoding the style, applying simple opacity, let's say 0.7?
> 
> [@retrofox](https://github.com/retrofox) No hardcoding. I mean using the Slider color setting. The green color in your example in [#51821 (comment)](https://github.com/woocommerce/woocommerce/issues/51821#issuecomment-2451635209).

Addressed [here](https://github.com/woocommerce/woocommerce/pull/52507/commits/45106fd16c51c1936e3374c45bc410f4866d676a)

**Comment by retrofox** [comment]
Where should the colors defined by the `Text & Icon` option be applied?

<img src="https://github.com/user-attachments/assets/052d1ba7-0c8e-4ce2-9c31-c415fda4eb04" width="600px" />

I guess the heading blocks composition. Above, the yellow text color

**Comment by roykho** [comment]
Closed via https://github.com/woocommerce/woocommerce/pull/52507
